JuliaExperimental/.gitignore

```diff
@@ -18,46 +18,31 @@
 /doc/*.tex
 /doc/*.toc
 /doc/manual.pdf
-/doc/NormalizInterface-bib.xml.bib
```

Comment:
So right now there are three .gitignore files. OK, it makes some sense that each of the GAP package has its own, but then the one on the global scope could / should be trimmed down a lot.

Or you merge them into a single .gitignore at the root, which then replaces rules like `/doc/*.tex` by `doc/*.tex`